Asthma/TabBarControllers/Dashboard/APHAirQualityDataModel.m

```diff
@@ -275,42 +273,48 @@ - (CLLocationManager*) locationManager
 - (void)locationManager:(CLLocationManager *) __unused manager
      didUpdateLocations:(NSArray *)locations
 {
+
     CLLocation * currentLocation = [locations lastObject];
     CLLocationCoordinate2D coordinate = [currentLocation coordinate];
     
     float lat = coordinate.latitude;
     float lon = coordinate.longitude;
     
-    NSTimeInterval lastAQICheckedTime = [[NSUserDefaults standardUserDefaults] doubleForKey:kAQILastChecked];
-    NSTimeInterval currentTime = [[NSDate date] timeIntervalSinceReferenceDate];
+    NSDate *lastAQICheckedTime = [[NSUserDefaults standardUserDefaults] objectForKey:kAQILastChecked] ?: nil;
+    
+    NSDate *currentTime = [NSDate date];
     self.aqiResponse = [[NSMutableDictionary alloc]init];
     
     __weak APHAirQualityDataModel *weakSelf = self;
     
-    if (!self.fetchingAirQualityReport && ((currentTime - lastAQICheckedTime) >= kAQICheckInterval)) {
+    if (!self.fetchingAirQualityReport && (([currentTime timeIntervalSinceDate: lastAQICheckedTime]) >= kAQICheckInterval)) {
+        [[NSUserDefaults standardUserDefaults]setObject:[NSDate new] forKey:kAQILastChecked];
+        [[NSUserDefaults standardUserDefaults]synchronize];
```

Comment:
Fix up spacing throughout this patch? add a space between "]" and "synchronize".